static/app/views/releases/detail/overview/sidebar/thresholdStatuses.tsx

```diff
@@ -0,0 +1,69 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import * as SidebarSection from 'sentry/components/sidebarSection';
+import {IconCheckmark, IconWarning} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {getExactDuration} from 'sentry/utils/formatters';
+
+import {Table, TableRow} from '../../../components/sideTable';
+import {ThresholdStatus} from '../../../utils/types';
```

Comment:
```suggestion
import {Table, TableRow} from 'sentry/components/sideTable';
import {ThresholdStatus} from 'sentry/utils/types';
```

Reply:
Oh - oof these aren't at the sentry level though.